snap7/client.py

```diff
@@ -8,11 +8,11 @@
 from datetime import datetime
 from typing import List, Optional, Tuple, Union
 
-import snap7
-from snap7.common import check_error, ipv4, load_library
-from snap7.types import S7SZL, Areas, BlocksList, S7CpInfo, S7CpuInfo, S7DataItem
-from snap7.types import S7OrderCode, S7Protection, S7SZLList, TS7BlockInfo, WordLen
-from snap7.types import S7Object, buffer_size, buffer_type, cpu_statuses, param_types
+from . import types
```

Comment:
sorry for the delay in review. Now we are at it, I think we should be consistent and then also import `block_types`, `S7CpuInfo` etc from `.types` and remove the `types.` prefix, OR remove all imports from `types` and append the module prefix everywhere.

Reply:
i think the first option is still the 'prettiest'.

Reply:
No worries for the delay. I'm would be more inclined to go with the prefix myself, but I will change the patch to go prefix-less later today.